mining/src/mempool/validate_and_insert_transaction.rs

```diff
@@ -14,21 +14,23 @@ use kaspa_consensus_core::{
 };
 use kaspa_core::info;
 
+use super::tx::{Orphan, Priority};
+
 impl<T: ConsensusMiningContext + ?Sized> Mempool<T> {
     pub(crate) fn validate_and_insert_transaction(
         &mut self,
         transaction: Transaction,
-        is_high_priority: bool,
-        allow_orphan: bool,
+        priority: Priority,
+        orphan: Orphan,
```

Comment:
Maybe struct name should be `Orphan` -> `Orphaning` ? not sure but not really important

Reply:
let's keep `Orphan` if you don't mind